releasenotes/notes/use-tls-1.2-by-default-e2983c21056c3359.yaml

```diff
@@ -0,0 +1,10 @@
+---
+enhancements:
+  - |
+    The ``min_tls_version`` configuration parameter is defined to `tlsv1.2` by
+    default. Set its value to `tlsv1.0` or `tlsv1.1` if a lower version is needed.
```

Comment:
```suggestion
    The ``min_tls_version`` configuration parameter now default to `tlsv1.2` instead of `tlsv1.0`.
    Set its value to `tlsv1.0` or `tlsv1.1` if a lower version is needed.
```